recipes/autouri/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "autouri" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 00caea88281a2e527e769619cab9c9ecf66baa553cc95203247e8d35526c708c
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```

Comment:
normally `python` and `pip` would be sufficient under `host`, but because of the import of the full package in the `setup.py` (https://github.com/ENCODE-DCC/autouri/blob/master/setup.py#L2), you're likely going to need all of the dependencies listed under `run`.

Reply:
Thanks, removed python version.